docs/installation/config/env_config.rst

```diff
@@ -0,0 +1,284 @@
+
+
+Initial superuser creation
+--------------------------
+
+A clean installation of Open Zaak comes without pre-installed or pre-configured admin
+user by default.
+
+Users of Open Zaak can opt-in to provision an initial superuser via environment
+variables. The user will only be created if it doesn't exist yet.
+
+* ``OPENZAAK_SUPERUSER_USERNAME``: specify the username of the superuser to create. Setting
+  this to a non-empty value will enable the creation of the superuser. Default empty.
+* ``OPENZAAK_SUPERUSER_EMAIL``: specify the e-mail address to configure for the superuser.
+  Defaults to ``admin@admin.org``. Only has an effect if ``OPENZAAK_SUPERUSER_USERNAME`` is set.
+* ``DJANGO_SUPERUSER_PASSWORD``: specify the password for the superuser. Default empty,
+  which means the superuser will be created *without* password. Only has an effect
+  if ``OPENZAAK_SUPERUSER_USERNAME`` is set.
+
+
+Advanced application server options
+-----------------------------------
+
+Open Zaak uses [uWSGI](https://uwsgi-docs.readthedocs.io/en/latest/Options.html) under
+the hood, which can be configured with a myriad of options. Most of these can be
+provided as environment variables as well. The following options below are a
+non-exhaustive list of options you may need with Open Zaak.
+
+* ``UWSGI_HTTP_TIMEOUT`` - defaults to 60s. If Open Zaak does not complete the request
+  within this timeout, then uWSGI will error out. This has been observed with certain
+  CMIS implementations causing slow requests where 60s is not sufficient.
+
+Celery
+------
+
+* ``CELERY_BROKER_URL``: the URL of the broker that will be used to actually send the notifications (default: ``redis://localhost:6379/1``).
+
+* ``CELERY_RESULT_BACKEND``: the backend where the results of tasks will be stored (default: ``redis://localhost:6379/1``)
+
+Cross-Origin-Resource-Sharing
+-----------------------------
+
+The following parameters control the CORS policy.
+
+* ``CORS_ALLOW_ALL_ORIGINS``: allow cross-domain access from any client. Defaults to ``False``.
+
+* ``CORS_ALLOWED_ORIGINS``: explicitly list the allowed origins for cross-domain requests.
+  Defaults to an empty list. Example: ``http://localhost:3000,https://some-app.gemeente.nl``.
+
+* ``CORS_ALLOWED_ORIGIN_REGEXES``: same as ``CORS_ALLOWED_ORIGINS``, but supports regular
+  expressions.
+
+* ``CORS_EXTRA_ALLOW_HEADERS``: headers that are allowed to be sent as part of the cross-domain
+  request. By default, ``Authorization``, ``Accept-Crs`` and ``Content-Crs`` are already
+  included. The value of this variable is added to these already included headers.
+  Defaults to an empty list.
+
+Initial configuration
+---------------------
+
+Open Zaak supports ``setup_configuration`` management command, which allows configuration via
+environment variables. 
+All these environment variables are described at :ref:`command line <installation_configuration_cli>`.
```

Comment:
If no string is specified, it will simply use the title of the page you're linking to (e.g. `Open Zaak configuration (CLI)`)
```suggestion
All these environment variables are described at :ref:`installation_configuration_cli`.
```